themes/digital.gov/layouts/topics/list.html

```diff
@@ -3,11 +3,250 @@
+              {{- end -}}
+            {{- else -}}
+              <h3>No Events to Display</h3>
+            {{- end -}}
+          </div>
+
+          <div class="grid-row grid-container-desktop">
+            {{/* Display first 5 news posts */}}
+            {{- $posts_stream := first 5 $posts.ByDate.Reverse -}}
+
+            {{- if $posts_stream -}}
+              <h2>{{ .Title }} news</h2>
+
+              {{- range $posts_stream -}}
+                {{- if .Params.source -}}
+                  {{/* Display external links, see layouts/news for news card template */}}
+                  {{- .Render "card-elsewhere" -}}
+                {{- else -}}
+                  {{ .Render "card-article" }}
+                {{- end -}}
+              {{- end -}}
+            {{- end -}}
+          </div>
+        </section>
+      {{ end }}
     {{ else }}
-      {{ partial "core/topics/topics-standard.html" . }}
+      <section class="usa-section">
+        <div class="grid-container grid-container-desktop">
+          {{/* Display events and news in one stream with pagination for topic default */}}
+          {{- if $stream -}}
+            <h2>
+              News and Events on
+              {{ .Title -}}
+            </h2>
+            <p>{{- len $posts }} posts</p>
+
+            {{- range $stream -}}
+              {{- if eq .Type "events" -}}
+                {{- .Render "card-event" -}}
+              {{- end -}}
+
+              {{- if eq .Type "news" -}}
+                {{/* Display external links, see layouts/news for news card template */}}
+                {{- if .Params.source -}}
+                  {{ .Render "card-elsewhere" }}
+                {{ else }}
+                  {{/* Internal links */}}
+                  {{ .Render "card-article" }}
+                {{ end }}
+              {{- end -}}
+            {{- end -}}
+          {{- end -}}
+
+          {{- partial "core/pagination.html" . -}}
+        </div>
+      </section>
     {{ end }}
+
+    {{ if .Params.legislation }}
+      {{/* Display remaining resources, events and news for topic collection */}}
+      {{ if lt .Paginator.PageNumber 2 }}
+        <section class="grid-container-desktop usa-section">
+          {{/* Only show these sections when on page 1, otherwise hide to show only More News and Events */}}
+          {{- $resource_settings := ( dict "header_title" (printf "More Resources on %s" .Title) "header_size" 2 "variant" "media" "list_header" 3) -}}
+          {{- partial "partials/core/collection/collection.html" (dict "collection_data" $resources "settings" $resource_settings ) -}}
+
+          {{- $services_settings := ( dict "header_title" "More Tools and Services" "header_size" 2 "variant" "media" "list_header" 3) -}}
+          {{- partial "partials/core/collection/collection.html" (dict "collection_data" $services "settings" $services_settings) -}}
+        </section>
+      {{ end }}
+      <div class="usa-section usa-section--light">
```

Comment:
Replaced `margin` and `padding` utility classes with `usa-section` for page sections.